doc/user/improve-visibility.md

```diff
@@ -0,0 +1,19 @@
+# How to improve the visibility of your project
+
+As a library author or maintainer, the visibility of your project on Scaladex is under your control.
+
+As soon as a release is pushed to Maven Central (Sonatype), Scaladex downloads the latest content and metadata from the Github repository. It then uses those metadata to index the project.
```

Comment:
We also download pom.xml from Maven Central (this is not directly related to SEO, but the way it is currently phrased can bring some confusion I think).